src/Components/CreateImageWizard/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -913,6 +914,60 @@ export const KernelList = () => {
   );
 };
 
+export const FirewallList = () => {
+  const firewall = useAppSelector(selectFirewall);
+
+  return (
+    <TextContent>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-v5-u-min-width"
+        >
+          Ports
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          <CodeBlock>
+            <CodeBlockCode>
+              {firewall.ports ? firewall.ports.join(' ') : 'None'}
```

Comment:
I feel like this was the intention. :thinking: 
```suggestion
              {firewall.ports.length ? firewall.ports.join(' ') : 'None'}
```

Reply:
Indeed :sweat_smile: updated!